bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusSlave.java

```diff
@@ -198,10 +200,9 @@ private void setCoil(Command command, ModbusBindingConfig config)
      * @throws ModbusException ModbusIOException on IO errors, ModbusSlaveException with protocol level exceptions
      * @throws ModbusUnexpectedTransactionIdException when response transaction id does not match the request
      */
-    protected void setRegister(Command command, ModbusBindingConfig config)
+    protected void setRegister(Command command, ModbusBindingConfig config, int readIndex, int writeRegister)

```

Comment:
You are right, they are. 
But to the RollerShutter item working (and to keep it as simple as possible inside the modbus-server configuration) I decide to use 2 consecutive write registers to handle UP/DOWN and STOP commands. Here it was mandatory to define a offset respectively.